flow/project.py

```diff
@@ -342,20 +342,20 @@ class FlowOperation(object):
 
     Any FlowOperation is associated with a specific command, which should be
     a function of :py:class:`~signac.contrib.job.Job`. The command (cmd) can
-    be stated as function, either by using str-substitution based on a job's
+    be stated as function, either by using string substitution based on a job's
     attributes, or by providing a unary callable, which expects an instance
     of job as its first and only positional argument.
 
     For example, if we wanted to define a command for a program called 'hello',
-    which expects a job id as its first argument, we could contruct the following
+    which expects a job id as its first argument, we could construct the following
     two equivalent operations:
 
     .. code-block:: python
 
         op = FlowOperation('hello', cmd='hello {job._id}')
         op = FlowOperation('hello', cmd=lambda 'hello {}'.format(job._id))
```

Comment:
This lambda needs an argument.
```suggestion
         op = FlowOperation('hello', cmd=lambda job: 'hello {}'.format(job._id))
```